web/docs/map-data-to-ocsf/README.md

```diff
@@ -254,23 +254,27 @@ Here's a template for the mapping pipeline:
 ```
 // (1) Move original event into dedicated field.
 this = { event: this }
-// (2) Computing new fields.
+// (2) Assign some intermediate values for use in the next step, e.g., because
+//     they're used multiple times.
 class_uid = 4001
+activity_id = 6
 ...
 // (3) Populate the OCSF event.
 this = {
   // --- Classification (required, optional, recommended) ---
```

Comment:
```suggestion
  // --- Classification ---
```

Unless there is a reason to specify this?

Reply:
Don't think so. I removed it.